cpp/include/cudf/quantiles.hpp

```diff
@@ -24,18 +24,23 @@ namespace cudf {
 namespace experimental {
 
 /**
- * @brief Computes a value for a quantile by interpolating between the values on
- *        either side of the desired quantile.
+ * @brief Computes values at desired quantiles by interpolating between the values on
+ *        either side of a desired quantile.
```

Comment:
Move detailed explanation out of @brief.

```suggestion
 *
 * Computes the specified quantiles by interpolating values between which they lie, using the interpolation
 * strategy specified in `interp`.
```